sdcm/stress_thread.py

```diff
@@ -106,6 +106,10 @@ def create_stress_cmd(self, node, keyspace_idx):
             self.profile = cs_profile
             self.keyspace_name = keyspace_name
 
+        if 'duration' in stress_cmd:
```

Comment:
should look for `duration=` the word duration is too general, can be in the name of the c-s use profile filename for example.

Reply:
fixed